src/algebra/algebra/basic.lean

```diff
@@ -474,6 +474,18 @@ lemma algebra_map_End_eq_smul_id (a : R) :
   ((algebra_map K (End K V)) a).ker = ⊥ :=
 linear_map.ker_smul _ _ ha
 
+namespace End
+
+variables {R} {M}
+
+/-- Construct a linear equivalence from a unit in `End R M`. -/
+@[simps] def linear_equiv_of_unit (f : units (End R M)) : M ≃ₗ[R] M :=
```

Comment:
This exists already as something like `general_linear_equiv`

Reply:
I removed the redundant code, but for some reason if I use `general_linear_equiv` instead of `to_linear_equiv` I get a typeclass resolution problem.